src/middlewares/rateLimit.middleware.ts

```diff
@@ -0,0 +1,55 @@
+import { Request, Response, NextFunction } from 'express';
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import * as rateLimit from 'express-rate-limit';
+import { ConfigService } from '@nestjs/config';
+import { RequestHandler } from '@nestjs/common/interfaces';
+
+const getMinutes = (minutes: number) => minutes * 60 * 1000;
+
+@Injectable()
+export class DefaultRateLimitMiddleware implements NestMiddleware {
+  private configuredMiddleware: RequestHandler;
+
+  constructor(private configService: ConfigService) {
+    const defaultWindowMs = this.configService.get(
```

Comment:
The variable is named a bit weird, `Min` instead of `Ms` would be correct. (Same for the Auth limiter below)